src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final String CIRCLE_COLOR = "WHITE";
+    private static final int CIRCLE_RADIUS = 10;
```

Comment:
These constants `CIRCLE_COLOR` and `CIRCLE_RADIUS` are specific to the creation of a default Circle figure and might not be needed as class-level constants if they are only used in the `getDefaultFigure()` method.